docs/cli/minercli.mdx

```diff
@@ -0,0 +1,240 @@
+---
+id: minercli
+sidebar_label: miner CLI
```

Comment:
Can we capitalize miner here to follow the standard practice for the docs.

Reply:
Done.